lib/article_json/export/amp/elements/text.rb

```diff
@@ -0,0 +1,44 @@
+module ArticleJSON
+  module Export
+    module AMP
+      module Elements
+        class Text < Base
```

Comment:
Isn't this exactly the same as the HTML text element? Maybe we could just extend it?